CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### Payments
+[FIXED][9011](https://github.com/stripe/stripe-android/pull/9011) Rebuild CardBrandView using native views instead of compose to prevent crashes when used with React Native and base android `Activity`.
```

Comment:
```suggestion
* [FIXED][9011](https://github.com/stripe/stripe-android/pull/9011) Fixed an issue where rendering Card Element in React Native and the base `Activity` would cause a crash.
```